CONTRIBUTING.rst

```diff
@@ -10,10 +10,10 @@ design enhancements issues as well as pull Requests to make *coala* more
 convenient, functional and bug free.
 
 To begin contributing have a look at the open issues labeled with
-difficulty/newcomer - `newcomer issues <https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`__.
+difficulty/newcomer - `newcomer issues <https://coala.io/newcomers>`__.
```

Comment:
wrong link, did you take a look at the diff and tried out the links? The next link isn't the same as before as well :(